site/app/models/gradeable/Gradeable.php

```diff
@@ -739,26 +752,30 @@ public function anySubmissions() {
             // Until we find a submission, assume there are none
             $this->any_submissions = false;
             if ($this->type === GradeableType::ELECTRONIC_FILE) {
-                $semester = $this->core->getConfig()->getSemester();
-                $course = $this->core->getConfig()->getCourse();
-                $submission_path = "/var/local/submitty/courses/" . $semester . "/" . $course . "/" . "submissions/" . $this->getId();
+                $submission_path = FileUtils::joinPaths(
+                    '/var/local/submitty/courses',
```

Comment:
Use the config variable to get the `/var/local/submitty`. Don't hardcode this in.